src/CMakeLists.txt

```diff
@@ -418,6 +423,91 @@ add_library(Halide
             # Including these as sources works around the need to "install" Halide_initmod
             $<TARGET_OBJECTS:Halide_initmod>
             $<TARGET_OBJECTS:Halide_c_templates>)
+
+##
+# Flatbuffers and Serialization dependencies.
+##
+
+# Build serialization, enabled by default
+option(WITH_SERIALIZATION "Include experimental Serialization/Deserialization code" ON)
+
+# flatbuffers is small and compiles quickly, but if you want/need to use
+# a local version (via find_package), configure with FLATBUFFERS_USE_FETCHCONTENT=OFF
+option(FLATBUFFERS_USE_FETCHCONTENT "Enable to download the Flatbuffers library via FetchContent" ON)
```

Comment:
This whole FetchContent thing makes me sad. We'll (I'll) be able to do _much_ better with CMake 3.24+ and its tighter integration between the two methods.

Reply:
Actually, an upgrade would give us so much leverage that I think we should poll our users to see if they can tolerate an early upgrade to 3.24.

Reply:
SGTM